modAionImpl/src/org/aion/zero/impl/cli/Cli.java

```diff
@@ -51,17 +52,50 @@
  */
 public class Cli {
 
-    File keystoreDir = new File(
-        System.getProperty("user.dir") + File.separator + CfgSsl.SSL_KEYSTORE_DIR);
+    private static final String BASE_PATH = System.getProperty("user.dir");
+
+    private static String BASE_PATH_WITH_NETWORK = BASE_PATH  + "/config/" + CfgAion.getNetwork();
```

Comment:
Isn't this the location where the database, keystore and so on will be created? Why add the config folder in between?

Reply:
this variable tracks which config file to copy from;
```
/home/aion/config/mainnet/config.xml
OR
/home/aion/config/conquest/config.xml
```